x-pack/legacy/plugins/siem/public/components/open_timeline/helpers.ts

```diff
@@ -58,7 +58,7 @@ export const isUntitled = ({ title }: OpenTimelineResult): boolean =>
 const omitTypename = (key: string, value: keyof TimelineModel) =>
   key === '__typename' ? undefined : value;
 
-const omitTypenameInTimeline = (timeline: TimelineResult): TimelineResult =>
+export const omitTypenameInTimeline = (timeline: TimelineResult): TimelineResult =>
```

Comment:
FYI this function and omitTypename look to exist in two places.

<img width="1329" alt="Screen Shot 2020-01-14 at 10 46 27 AM" src="https://user-images.githubusercontent.com/1151048/72368371-2ffb3d00-36bb-11ea-8750-f0f6a157c066.png">